src/core/components/forms/radioCard/radioCard.test.tsx

```diff
@@ -72,4 +72,21 @@ describe('<RadioCard/> component', () => {
 
         expect(screen.getByRole('radio')).toHaveValue(value);
     });
+
+    it('renders children when radio button is checked', async () => {
+        const user = userEvent.setup();
+        render(
+            createTestComponent({
+                children: <div data-testid="child-content">child content</div>,
+            }),
+        );
```

Comment:
We can just pass a text as children and avoid multiple line-breaks
```suggestion
        const user = userEvent.setup();
        
        const children = 'test-children';
        render(createTestComponent({ children }));
```